Dockerfile

```diff
@@ -1,67 +1,17 @@
-FROM registry.access.redhat.com/ubi8/nodejs-20:latest AS BUILD_IMAGE
-
-### BEGIN REMOTE SOURCE
-# Use the COPY instruction only inside the REMOTE SOURCE block
-# Use the COPY instruction only to copy files to the container path $REMOTE_SOURCES_DIR/activemq-artemis-self-provisioning-plugin/app
-ARG REMOTE_SOURCES_DIR=/tmp/remote_source
-RUN mkdir -p $REMOTE_SOURCES_DIR/activemq-artemis-self-provisioning-plugin/app
-WORKDIR $REMOTE_SOURCES_DIR/activemq-artemis-self-provisioning-plugin/app
-# Copy package.json and yarn.lock to the container
-COPY package.json package.json
-COPY yarn.lock yarn.lock
-ADD . $REMOTE_SOURCES_DIR/activemq-artemis-self-provisioning-plugin/app
-RUN command -v yarn || npm i -g yarn
-### END REMOTE SOURCE
-
+FROM registry.access.redhat.com/ubi8/nodejs-16:latest AS build
 USER root
+RUN command -v yarn || npm i -g yarn
 
-## Set directory
-RUN mkdir -p /usr/src/
-RUN cp -r $REMOTE_SOURCES_DIR/activemq-artemis-self-provisioning-plugin/app /usr/src/
+ADD . /usr/src/app
 WORKDIR /usr/src/app
+RUN yarn install && yarn build
```

Comment:
could you update here with the title in comment lines before the Command. i.e
```
## Install dependencies and Build application
RUN yarn install && yarn build
```

Reply:
I'm not sure the comment is really necessary as it only paraphrases the commands that executed under, which are install and build. Comments should always bring something more to the table than just stating what is done.